pkgs/development/python-modules/databases/default.nix

```diff
@@ -13,42 +14,48 @@
 
 buildPythonPackage rec {
   pname = "databases";
-  version = "0.4.3";
-  disabled = isPy27;
+  version = "0.5.3";
+
+  disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "encode";
     repo = pname;
     rev = version;
-    sha256 = "0aq88k7d9036cy6qvlfv9p2dxd6p6fic3j0az43gn6k1ardhdsgf";
+    sha256 = "sha256-67ykx7HKGgRvJ+GRVEI/e2+x51kfHHFjh/iI4tY+6aE=";
   };
 
   propagatedBuildInputs = [
-    aiocontextvars
-    sqlalchemy
-  ];
-
-  checkInputs = [
+    aiopg
     aiomysql
     aiosqlite
     asyncpg
     pymysql
+    sqlalchemy
+  ] ++ lib.optionals (pythonOlder "3.7") [
```

Comment:
What does exactly `pythonOlder` means?

Reply:
Hm. Probably! I just cherry-picked this commit from https://github.com/NixOS/nixpkgs which might have a bit of a bug.